spec/services/list_languages_taught_in_modern_language_course_spec.rb

```diff
@@ -0,0 +1,59 @@
+describe ListLanguagesTaughtInModernLanguageCourse do
+  describe '#call' do
+    context "when the course isn't for modern languages" do
+      let(:course_name) { 'Physical Education with French' }
+
+      context 'with two language subjects' do
+        let(:ucas_subject_names) do
+          'French, German, Languages, Languages (European), Secondary'
+        end
+
+        subject { described_class.call(course_name, ucas_subject_names) }
+
+        it { should eq([]) }
+      end
+    end
+
+    context "when the course name contains MFL" do
```

Comment:
subject mapper dont really touch course name for language subject

Reply:
I think this is what me and @fofr are discussing above.